src/public/services/__tests__/BillingService.test.ts

```diff
@@ -0,0 +1,52 @@
+import MockAxios from 'jest-mock-axios'
+
+import * as BillingService from '../BillingService'
+
+jest.mock('axios', () => MockAxios)
+
+describe('BillingService', () => {
+  describe('getBillingInfo', () => {
+    const MOCK_DATA = {
+      adminEmail: 'Big Mock',
```

Comment:
hahahahaah

Reply:
i aim to make reading tests joyful HAHAHAH